src/plugins/dashboard/public/dashboard_actions/filters_notification_popover_contents.tsx

```diff
@@ -60,16 +60,16 @@ export function FiltersNotificationPopoverContents({ context }: FiltersNotificat
   });
 
   return (
-    <>
+    <EuiForm
+      component="div"
+      css={css`
+        min-width: 300px;
+      `}
+    >
       {isLoading ? (
-        <EuiLoadingContent lines={3} />
+        <EuiSkeletonText lines={3} />
       ) : (
-        <EuiForm
-          component="div"
-          css={css`
-            min-width: 300px;
-          `}
-        >
+        <>
```

Comment:
I moved the `EuiForm` up one level so that the popover doesn't jump in size quite as much when loading:

| Before | After |
|--------|-------|
| ![May-01-2023 16-34-48](https://user-images.githubusercontent.com/8698078/235733114-f935f759-fe29-4a61-a804-ef9c2bedfc21.gif) | ![May-01-2023 16-32-22](https://user-images.githubusercontent.com/8698078/235733097-04257fa5-128a-4f52-8c51-2ccacbe04314.gif) |